### ISSUE TITLE
Yandex Mail support?

### ISSUE THREAD

**Issue by pathintegral** [nylas-mail-lives/nylas-mail#124]
I'm very glad to hear that "nylas-mail-lives". Thanks for keeping Nylas Mail alive.

However, I can't sync my Yandex Mail. Connection is ok, I can see my yandex account on accounts but sync does not work. Is it a bug or unimplemented feature?

**Comment by dweremeichik** [comment]
Does yandex support IMAP ans SMTP? If so it should work. Pleas note that no version of Nylas Mail has ever supported POP.

**Comment by pathintegral** [comment]
Yes, it supports. I used same configurations (IMAP, SMTP) for Thunderbird, Mailspring etc. and there was no problem.

**Comment by seesemichaelj** [comment]
I'm guessing is an unimplemented/buggy protocol on either Yandex or our sync-engine (probably us given the other successful clients)

**Comment by pathintegral** [comment]
Are you willing to do something about that? It's ok for me to use browser for Yandex Mail until you solve this issue. 
Because Nylas Mail is so far best email client for linux as far as i saw.

**Comment by seesemichaelj** [comment]
Unfortunately, NML is mainly developed by the collective community rather than an organization. Thus, things really only get fixed when someone wants them fixed since this is essentially a side hobby project for everyone that contributes to it. I'd love to say "yes" but I'd have to quit my fulltime job to support everyone's issues :/

Mailspring is essentially the same as Nylas Mail Lives with a closed-source, but faster sync engine. It gets the kind of support you're looking for because there is a full time developer which is monetized on subscriptions.
